shared/protocol/src/main/java/io/pravega/shared/protocol/netty/WireCommands.java

```diff
@@ -1332,6 +1332,75 @@ public static WireCommand readFrom(DataInput in, int length) throws IOException
         }
     }
 
+    @Data
+    public static final class MergeSegmentsBatch implements Request, WireCommand {
+        final WireCommandType type = WireCommandType.MERGE_SEGMENTS_BATCH;
+        final long requestId;
+        final String target;
+        final List<String> sources;
+        @ToString.Exclude
+        final String delegationToken;
+
+        @Override
+        public void process(RequestProcessor cp) {
+            cp.mergeSegmentsBatch(this);
+        }
+
+        @Override
+        public void writeFields(DataOutput out) throws IOException {
+            out.writeLong(requestId);
+            out.writeUTF(target);
+            out.writeInt(sources.size());
+            for (int i = 0; i < sources.size(); i++) {
+                out.writeUTF(sources.get(i));
+            }
+            out.writeUTF(delegationToken == null ? "" : delegationToken);
+        }
+
+        public static WireCommand readFrom(DataInput in, int length) throws IOException {
+            long requestId = in.readLong();
+            String target = in.readUTF();
+            int sourceCount = in.readInt();
+            List<String> sources = new ArrayList<>(sourceCount);
+            for (int i = 0; i < sourceCount; i++) {
+                sources.add(in.readUTF());
+            }
+            String delegationToken = in.readUTF();
+            return new MergeSegmentsBatch(requestId, target, sources, delegationToken);
+        }
+    }
+
+    @Data
+    public static final class MergeTableSegments implements Request, WireCommand {
```

Comment:
Remove MergeTableSegments and SealTableSegments

Reply:
fixed